schutzbot/deploy.sh

```diff
@@ -21,6 +21,12 @@ function retry {
 # Get OS details.
 source /etc/os-release
 
+# Register RHEL if we are provided with a registration script.
+if [[ -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
+    sudo chmod +x $RHN_REGISTRATION_SCRIPT
```

Comment:
How come the script does not already have `+x`?

Reply:
Jenkins dumps it on the host as a secret, so it's dumped as a plain file without `-x` set.